pkg/agent/api/delegatedidentity/v1/service.go

```diff
@@ -108,6 +116,8 @@ func (s *Service) isCallerAuthorized(ctx context.Context, log logrus.FieldLogger
 func (s *Service) SubscribeToX509SVIDs(req *delegatedidentityv1.SubscribeToX509SVIDsRequest, stream delegatedidentityv1.DelegatedIdentity_SubscribeToX509SVIDsServer) error {
 	ctx := stream.Context()
 	log := rpccontext.Logger(ctx)
+	latency := adminapi.StartFirstUpdateLatency(s.metrics)
```

Comment:
I think we want this to be the first method call in this handler method so we're capturing the full execution time of the handler